include/openenclave/enclave.h

```diff
@@ -173,6 +173,21 @@ void* oe_host_realloc(void* ptr, size_t size);
  */
 void* oe_host_calloc(size_t nmemb, size_t size);
 
+/**
+ * Fill block of host memory.
+ *
+ * This function sets the first **num** bytes of the block of host memory
```

Comment:
Mentions of parameter names should use /p instead of **

See https://stackoverflow.com/questions/15398711/whats-the-right-way-to-reference-a-parameter-in-doxygen